sources/packages/backend/apps/workers/src/main.ts

```diff
@@ -3,10 +3,12 @@ import { NestFactory } from "@nestjs/core";
 import { LoggerService } from "@sims/utilities/logger";
 import { WorkersModule } from "./workers.module";
 import { ZeebeTransportStrategy } from "./zeebe/zeebe-transport-strategy";
+import { ConfigService } from "@sims/utilities/config";
 import { SystemUsersService } from "@sims/services";
 
 (async () => {
   const workers = await NestFactory.create(WorkersModule, { bufferLogs: true });
+  const config = workers.get<ConfigService>(ConfigService);
```

Comment:
I am aware that this was probably copied but `workers.get(ConfigService)` should be enough.